### PR TITLE
taxonomy: correction cranberry lingonberry

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**
### Related issue(s) and discussion

- Fixes #[ISSUE NUMBER]

### CODE REVIEW

**Review by alexgarel — APPROVED** [review]
Thanks

File: taxonomies/food/ingredients.txt [hunk 1]
```diff
@@ -43499,12 +43498,12 @@ pl:sok z koncentratu z kiwi
...
 da:tyttebær
-de:Preiselbeeren
-es:Arándanos agrios,arándanos
+de:Preiselbeere, Preiselbeeren
+es:Arándanos agrios
 et:pohl
 fi:Puolukka
-fr:myrtille rouge
-it:Mirtilli rossi
+fr:airelle rouge
```
Comment (stephanegigandet) [inline taxonomies/food/ingredients.txt:43505]:
```suggestion
fr:airelle rouge, airelles rouges
```

**Review by stephanegigandet — CHANGES_REQUESTED** [review]
@moon-rabbitOFF We have about one hundred products that list "airelles" as one of their ingredients: https://fr.openfoodfacts.org/cgi/search.pl?search_terms=airelles&search_simple=1&action=process

It seems that in most cases, "airelles" is used for "cranberries" / "canneberges", so I think it would make sense to keep it as a synonym of en:cranberry

File: taxonomies/food/ingredients.txt [hunk 2]
```diff
@@ -43517,6 +43516,10 @@ sv:lingon
 # 7 products @2018-10-01
 eurocode_2_group_3:en: 9.30.44
 
+<en:lingonberry
+en:wild lingoberry
```
Comment (stephanegigandet) [inline taxonomies/food/ingredients.txt:43520]:
```suggestion
en:wild lingonberry, wild lingonberries
```

**Review by stephanegigandet — APPROVED** [review]
Thanks!

**Pull request merged**